File: java/src/test/java/codingdojo/CalcStats2Test.java
package codingdojo;

import org.junit.jupiter.api.Test;

import java.util.stream.Stream;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class CalcStats2Test {
    @Test
    void count() {
        Stream<Integer> values = Stream.of();
        long count = CalcStats2.count(values.toList());
        assertEquals(0, count);
    }
}

File: php/test/CalcStats2Test.php
<?php

namespace Tests;

use CodingDojo\CalcStats2;
use PHPUnit\Framework\TestCase;

class CalcStats2Test extends TestCase
{
    public function testCount()
    {
        $values = [];
        $count = CalcStats2::count($values);
        $this->assertEquals(0, $count);
    }
}
